opennlp-tools/src/test/java/opennlp/tools/eval/OntoNotes4NameFinderEval.java

```diff
@@ -86,18 +85,8 @@ private static void crossEval(TrainingParameters params, String type, double exp
   }
 
   @BeforeClass
-  public static void verifyTrainingData() throws IOException {
-    MessageDigest digest = EvalUtil.createDigest();
-
-    try (ObjectStream<NameSample> samples = createNameSampleStream()) {
-      NameSample sample;
-      while ((sample = samples.read()) != null) {
-        digest.update(sample.toString().getBytes(StandardCharsets.UTF_8));
-      }
-
-      Assert.assertEquals(new BigInteger("168206908604555450993491898907821588182"),
-          new BigInteger(1, digest.digest()));
-    }
+  public static void verifyTrainingData() throws Exception {
+    verifyTrainingData(createNameSampleStream(), new BigInteger("168206908604555450993491898907821588182"));
```

Comment:
To handle this better we should introduce a method that can verify certain files recursively in a directory.